internal/scheduler/scheduler.go

```diff
@@ -221,6 +234,11 @@ func (s *Scheduler) cycle(ctx context.Context, updateAll bool, leaderToken Leade
 			return err
 		}
 
+		// Report counts of scheduled jobs per queue.
+		// TODO: preemptible jobs, possibly other metrics here.
+		s.metrics.ReportScheduledJobs(overallSchedulerResult.ScheduledJobs)
```

Comment:
we probably only want to set these if  `shouldSchedule && leaderToken.leader`

Reply:
hmm I see what you mean. I thought it would exit out earlier if not leader but I guess that neglects the fact that it could lose leadership partway through scheduling.